src/main/java/com/epam/wl/services/TestLoginService.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.wl.services;
+
+import com.epam.wl.DBHelper;
+import com.epam.wl.dao.UserDAO;
+import com.epam.wl.entities.User;
+import com.epam.wl.enums.UserRole;
+
+import java.sql.SQLException;
+import java.util.Optional;
+
+public class TestLoginService {
+    public static UserDAO userDAO = new UserDAO(DBHelper.getEmbeddedDatabase());
```

Comment:
This field should be private final

Reply:
Making this field non-static entails internal server error in case if I create a new instance of this class